test/basic-images.test

```diff
@@ -213,6 +213,17 @@ test_expect_success mkfs_jffs2 "jffs2" "
 	md5sum -c '${testdir}/jffs2.md5'
 "
 
+exec_test_set_prereq qemu-img
+test_expect_success "qemu" "
+	setup_test_images &&
+	run_genimage qemu.config test.qcow &&
+	check_size images/test.qcow 196608 &&
+	qemu-img check images/test.qcow &&
+	qemu-img info images/test.qcow &&
+	qemu-img info images/test.qcow | diff -q '${testdir}/qemu.info' - &&
```

Comment:
Unfortunately, this does breaks with new qemu versions. Try this:
<pre>
qemu-img info images/test.qcow | \
        sed -e '/ *refcount bits:/d' -e '/ *corrupt:/d' > qemu.info &&
test_cmp '${testdir}/qemu.info' qemu.info
</pre>

Reply:
Something went wrong during rebase, please switch to #29

Reply:
@AlexandreFournier you can also just use `git push --force` to repair a wrong push.

Reply:
I know, I thought something was broken in my local repository ```git rebase -i``` was rewriting the last 4 commits even when untouched. At the end I decided to re-fork and clone. But the problem is still here... strange. So thanks, but I think it's too late, isn't it ?